.rubocop.yml

```diff
@@ -166,3 +166,5 @@ AllCops:
     - 'tmp/**/*'
     - 'storage/**/*'
     - 'db/migrate/20230426130150_init_schema.rb'
+    # added this because of bulk insert operation present in the file
+    - 'lib/test_data/contact_batch_service.rb'
```

Comment:
if is a specific cop breaking in a file and we can't figure out a fix lets do inline disable instead 
 ref: https://github.com/chatwoot/chatwoot/blob/ae0b68147e3a8a4a7effda9888c939f680f949f6/app/builders/messages/facebook/message_builder.rb#L129